etl/data_helpers/population.py

```diff
@@ -57,9 +65,12 @@ def add_population(
     pd.DataFrame
         Dataframe with extra column `population`.
     """
-    # Load granular population dataset
-    ds = Dataset(DATA_DIR / "garden" / "un" / "2022-07-11" / "un_wpp")
-    pop = ds["population_granular"].reset_index()
+    if ds_un_wpp is None:
+        ds_un_wpp_path = DATA_DIR / "garden" / "un" / "2022-07-11" / "un_wpp"
```

Comment:
Super Nit - when I search for steps (e.g. for refactoring), I usually their URI like `garden/un/2022-07-11/un_wpp`. If this was written as
```
ds_un_wpp_path = DATA_DIR / "garden/un/2022-07-11/un_wpp"
``` 
it would be easier to find. (Feel free to ignore this!)